src/Resources/contao/library/Contao/Widget.php

```diff
@@ -263,6 +265,22 @@ public function __set($strKey, $varValue)
 				$this->strPrefix = $varValue;
 				break;
 
+			case 'useRawRequestData':
+				if ($varValue === true)
+				{
+					/** @var Request $request */
+					$request = \System::getContainer()->get('request_stack')->getCurrentRequest();
+					$this->setInputCallback(function() use ($request) {
+						return $request->request->get($this->name);
```

Comment:
I don't think so. If you want the raw request data you need to handle the array yourself if you need to, no?